uPortal-api/uPortal-api-rest/build.gradle

```diff
@@ -14,4 +16,15 @@ dependencies {
 
     compileOnly "${portletApiDependency}"
     compileOnly "${servletApiDependency}"
+
+    testCompile "junit:junit:${junitVersion}"
+    testCompile "org.mockito:mockito-core:${mockitoVersion}"
```

Comment:
Removing junit from here would also resolve.

```
This project contains lint violations. A complete listing of the violations follows. 
Because some were serious, the overall build status has been changed to FAILED

error     duplicate-dependency-class         org.hamcrest:hamcrest-all:1.3 in configuration ':uPortal-api:uPortal-api-rest:testCompile' has 45 classes duplicated by org.hamcrest:hamcrest-core:1.3 (use --info for detailed class list) (no auto-fix available)
```

which is caused by `junit` without a `hamcrest-core` exclusion (top level project has been patched).